shared/src/packages/journal/libs/validation-schemas/create-journal-entry.validation-schema.ts

```diff
@@ -0,0 +1,30 @@
+import joi from 'joi';
+
+import {
+  JournalEntryValidationMessage,
+  JourneyEntryValidationRule,
+} from '../enums/enums.js';
+import { type JournalEntryCreateRequestDto } from '../types/types.js';
+
+const createJournalEntryValidationSchema = joi.object<
```

Comment:
```suggestion
const createJournalEntry = joi.object<
```
and then  `createJournalEntry as createJournalEntryValidationSchema` in re-export